examples/go/basic_example/server.go

```diff
@@ -0,0 +1,180 @@
+/**
+ * Implements the qlik.sse.Connector service.
+ */
+
+//go:generate protoc -I ../../../proto ../../../proto/ServerSideExtension.proto --go_out=plugins=grpc:./gen
+
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net"
+
+	"github.com/golang/protobuf/proto"
+
+	pb "github.com/qlikmats/server-side-extension/examples/go/basic_example/gen"
```

Comment:
Shouldn't the path be relative rather than point out your fork repo?

Reply:
Don't think it should be relative, if I have understood Go import paths correctly, but it should not point to my fork when "merging" to the parent repo. For now, I think it needs to be like this in order to work...